weatherman.py

```diff
@@ -146,6 +146,22 @@ def avg_mean_humid(self, by = None):
         else:
             return None
 
+
+    def draw_chart_for_month(self, by = None):
+        date = shortToLongDate(by)
+        year = date[0]
+        month = date[1]
+        for e in wmentries.entries.keys():
+            if by is not None and (year not in e or month not in e):
+                continue
+            for d in wmentries.entries[e]:
```

Comment:
avoid using variables like **e, d, by try** to use something meaningful

Reply:
I am guessing you are using dict here so better way to traverse dict in python is something like 
```
for key, value in wmentries.entries.iteritems()
```
and look into iteritems() as well.

Reply:
ok